README.md

```diff
@@ -161,6 +161,90 @@ If `feature_auth_required` is enabled keycloak will be installed and a random pa
 To view these credentials:
 `oc -n konveyor-tackle extract secret/tackle-keycloak-sso --to=-`
 
+## Enable KAI (Solution Server)
+
+KAI is an optional, experimental “solution server” that integrates with the Hub. To enable it:
+
+1. Update the Tackle CR to enable KAI
+
+```
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tackle
+  namespace: konveyor-tackle
+spec:
+  experimental_deploy_kai: true
+```
+
+2. Create a credentials secret named `kai-api-keys` in the same namespace. Choose the provider you will use:
+
+- IBM GenAI (default provider `ChatIBMGenAI`)
+```
+kubectl create secret generic kai-api-keys -n konveyor-tackle \
```

Comment:
I think these secret keys are actually not accurate anymore, it should now be ENVVAR_NAME: VALUE
ie OPENAI_API_KEY: sk....

Reply:
The environment variables will just be injected directly into the container so it is provider dependent what it will look for

Reply:
Maybe what you mean is that, my copying from previous examples works in practice but isn't applicable to all of the providers?

Reply:
I'm saying I wouldn't expect those variables to work (at least once hint generation began), since variables are just injected as-is from the secret now

Reply:
I think I have it right now.